### DISCUSSION TITLE
chatgpt secret key file

### DISCUSSION THREAD

**Discussion by PlumpMath** [gvolpe/neovim-flake#16] [Q&A]
Recently, the nix-config settings have changed a lot, and a ChatGPT part has been added. The configuration installs fine when I disable the ChatGPT part, but...

As a computer security novice, I have never tried uploading encrypted files to Git. Following your setup, I have looked into git-crypt and roughly understood what it is. However, it does not seem like git-crypt alone will satisfy your configuration requirements. If there are any additional steps needed, do I need to perform OpenSSL processing to encrypt the secret files? And, do I need any other settings like GPG?

I am also curious about the following code:

```
 encryptedSha256  = "blah blah";
 emptyValue = "secret";
```

Thanks.

**Comment by dickmao** [comment]
@PlumpMath You should switch back to Yuna.  She is hotter than Nabong.

**Reply by PlumpMath** [reply]
Wa~ ^^ How did you know Nayeon's nickname? Nabong is Tá bom. dickmao! ㅍㅎㅎㅎ Did you switch from Nabong? Yuna is pretty, but Nayeon is also disgustingly cute. 
...
I had something I wanted to tell you... but I never thought we'd meet in a place like this..
Thank you for suddenly showing up. (Appreciation for NaYeon Yuna)
Can't you create something like a Nix overlay for commercial-emacs so that it can be selected by default in nixpkgs? 
Take the code from @shaunsingh, this friend. so I used to compile it diligently every day... 
but I gave up a few months ago. I want to go back.

I wish I could choose both neovim-flake and commercial-emacs by default in Nixpkgs

**Reply by PlumpMath** [reply]
https://www.youtube.com/shorts/Du18tYKAZLM
